### PR TITLE
Ironbank now supports EPAS operands

### PR DESCRIPTION
**Pull request by jpe442**
[DOCS-776](https://enterprisedb.atlassian.net/browse/DOCS-776)

- Made mention that EPAS operands can now be pulled and used from the Iron Bank
- Added a section for deploying with the downloaded operand image

[DOCS-776]: https://enterprisedb.atlassian.net/browse/DOCS-776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx [hunk 1]
```diff
@@ -33,19 +33,25 @@ Specifically, you will need to use your
...
-To find the desired operator image, we recommend to use the search tool to look
+To find the desired operator or operand images, we recommend to use the search tool to look
 with the string `edb`, and filter by `Tags`, looking for `stable`, as shown in
-the image. From there, you can get the instruction to pull the image, for
-example using Docker:
+the image. From there, you can get the instruction to pull the image:
 
 ![pulling-ironbank-images](./images/ironbank/pulling-the-image.png)
 
+For example, to pull the latest EPAS16 operand from Ironbank, you can run:
```
Comment (stsantilena) [inline product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx:42]: Minor nit: this is explicitly pulling version 16 of EPAS. In Docker `latest` is a specific tag the always pulls the latest version.

File: product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx [hunk 2]
```diff
@@ -90,3 +96,33 @@ directly to your Kubernetes nodes.
 
 Once you have this in place, you can apply your manifest normally with
 `kubectl apply -f`, as described in the [installation instructions](/postgres_for_kubernetes/latest/installation_upgrade/).
+
+## Deploying clusters with EPAS operands using their Iron Bank image
+
+To deploy a cluster using an [operand](/postgres_for_kubernetes/latest/private_edb_registries/#operand-images) from a pulled image, you must reference the Ironbank operand image appropriately in a Custom Resource (CR) file.
```
Comment (stsantilena) [inline product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx:102]: Suggest "To deploy a cluster using the EPAS operand, you must..."

File: product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx (hunk 1 above)
Comment (jpe442) [reply product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx:42]: Yep, I say the latest version of EPAS16 not of EPAS, so I am not trying to token `latest` here.

Comment (jpe442) [reply product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx:42]: I went ahead and removed the word "latest" just to minimize confusion.

**Review by NiccoloFei — APPROVED** [review]

File: product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx [hunk 3]
```diff
@@ -90,3 +95,33 @@ directly to your Kubernetes nodes.
...
+
+1. Create or edit a `Cluster` resource YAML file with the following content:
+
+  ```yaml
+  apiVersion: postgresql.k8s.enterprisedb.io/v1
+  kind: Cluster
+  metadata:
+    name: cluster-example-full
+  spec:
+    imageName: registry1.dso.mil/ironbank/enterprisedb/edb-postgres-advanced-16@16
```
Comment (gvasquezvargas) [inline product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx:112]: When we provide an example to pull the images above, we use the format: 
`registry1.dso.mil/ironbank/enterprisedb/edb-postgres-advanced-16:16`

But here, we say:
```suggestion
    imageName: registry1.dso.mil/ironbank/enterprisedb/edb-postgres-advanced-16@16
```

Not sure if it is a typo that one is with 16:16 and the other with 16@16.
If it is not a typo, maybe we should call out the difference so the reader know it is correct this way?

**Review by gvasquezvargas — APPROVED** [review]
Approved, left a minor comment, not sure if a typo or a quirk.

File: product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx (hunk 3 above)
Comment (jpe442) [reply product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx]: Fixed it.

**Pull request merged**